src/main/java/me/botsko/prism/commands/WandCommand.java

```diff
@@ -250,19 +232,47 @@ else if (type.equalsIgnoreCase("off")) {
                     // it
                     // was
                     // successful
-                    call.getPlayer().sendMessage(Prism.messenger.playerError("Notice: Only some parameters used.."));
+                    Prism.messenger.sendMessage(call.getPlayer(),
+                            Prism.messenger.playerError(Il8nHelper.getMessage("wand-params-few")));
                 }
             }
 
             // Store
             Prism.playersWithActiveTools.put(call.getPlayer().getName(), wand);
         } else {
-            if (oldwand != null) {
-                oldwand.disable(call.getPlayer());
+            if (oldWand != null) {
+                oldWand.disable(call.getPlayer());
             }
         }
     }
 
+    static void sendWandStatus(final CommandSender sender,
+                               @PropertyKey(resourceBundle = "languages.message") String wandStatusMessageKey,
+                               final boolean status,
+                               final String wandType,
+                               final String parameters) {
+        final TextComponent state;
+        if (status) {
+            state = Il8nHelper.getMessage("enabled").color(NamedTextColor.GREEN);
+        } else {
+            state = Il8nHelper.getMessage("disabled").color(NamedTextColor.RED);
+        }
+        TextComponent out = Prism.messenger
+                .playerHeaderMsg(Il8nHelper.getMessage(wandStatusMessageKey)
+                        .replaceText(Pattern.compile("<status>"),
+                                builder -> Component.text().append(state)));
+        if (status) {
+            out.append(Component.newline())
+                    .append(Il8nHelper.getMessage("wand-item-type")
+                            .replaceText(Pattern.compile("<itemType>"),
+                                    builder -> Component.text().content(wandType))
+                            .replaceText(Pattern.compile("<parameters"),
+                                    builder -> Component.text().content(parameters)));
```

Comment:
Codacy found an issue: ['lambda arguments' has incorrect indentation level 36, expected level should be one of the following: 32, 34.](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=6279951)